lib/Subscriber/Behavior/Audit/BlameSubscriber.php

```diff
@@ -92,9 +92,14 @@ public function handlePersist(PersistEvent $event)
             return;
         }
 
-        $node = $event->getNode();
         $locale = $event->getLocale();
 
+        if (!$locale) {
+            return;
```

Comment:
I didn't do that in the first place, and then there was a properties called like `i18n:-changer`, so I would say this is necessary. And even if it would be it should not update any data from a language which was not really edited.